core/config/project_settings.cpp

```diff
@@ -645,9 +642,7 @@ Error ProjectSettings::_setup(const String &p_path, const String &p_main_pack, b
 		return err;
 	}
 
-	if (resource_path.length() && resource_path[resource_path.length() - 1] == '/') {
-		resource_path = resource_path.substr(0, resource_path.length() - 1); // Chop end.
-	}
+	resource_path = resource_path.trim_suffix("/");
```

Comment:
Old code says that if the resource path isn't empty and the last character is '/', then we should remove that last character from the string. `trim_suffix()` can do the same.